docs/astro/src/content/docs/guide/development/translations.mdx

```diff
@@ -1,10 +1,11 @@
 ---
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+// cSpell: ignore msgfmt msgctx msginit poedit Lokalize Transifex 
```

Comment:
Ahh, that's where to place the cSpell "spell" in MDX files. TIL :)

Reply:
Also did you notice how to turn on or off the next / previous buttons at the bottom of certain pages? It's a shame that behaviour isn't automatic but the grouping a 3rd party plugin, not inbuilt Starlight.

Reply:
I thought that’s kind of neat ;-)